.github/workflows/build.yml

```diff
@@ -30,3 +30,21 @@ jobs:
         uses: qard/heaviest-objects-in-the-universe@v1
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
+  coverage:
+    runs-on: ubuntu-latest
+    env:
+      CARGO_TERM_COLOR: always
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Rust
+        run: rustup update nightly
+      - name: Install cargo-llvm-cov
+        uses: taiki-e/install-action@cargo-llvm-cov
+      - name: Generate code coverage
+        run: cargo +nightly llvm-cov --all-features --workspace --lcov --output-path lcov.info
```

Comment:
Does this run the tests? How does it figure out the coverage?

Also, do we even need Rust tests? Since this is a JavaScript binding, would just relying on JavaScript tests be enough?

Reply:
Since this is just a binding on top of another library, I don't think we would (or should) need that. Especially since the pipeline is set up in a way that expects the compiled binary to be used for testing, and Cargo tests require compiling in-place with different settings than the final binary.